torchvision/transforms/functional.py

```diff
@@ -796,3 +796,20 @@ def to_grayscale(img, num_output_channels=1):
         raise ValueError('num_output_channels should be either 1 or 3')
 
     return img
+
+
+def erase(img, x, y, h, w, value=0):
+    """ Erase the given Image with given value.
+        'Random Erasing Data Augmentation' by Zhong et al.
+        See https://arxiv.org/pdf/1708.04896.pdf
+     Args:
+        img (Tensor): Image to be erased.
+        x, y: Upper left pixel coordinate.
+        w: Width of the erased image.
+        h: Height of the erased image.
+        value: Erasing value.
+     Returns:
+        Erased Image.
+    """
+    img[:, x:x + h, y:y + w] = value
```

Comment:
can you add an assert that `img` is an instance of a `torch.Tensor`?
```python
if not isinstance(img, torch.Tensor):
    raise TypeError("...")
```

Reply:
Thanks. I have added it.